2024-01-15 06:36:08,140 - root - INFO - Building Model ...
2024-01-15 06:36:08,147 - root - INFO - Start Training ...
2024-01-15 06:36:08,440 - root - INFO - Start Training: Epoch 1
168
Traceback (most recent call last):
  File "/opt/ml/input/code/dkt/train.py", line 43, in <module>
    main(args) # main함수 실행 -> 학습!!!!!!!!!!!
  File "/opt/ml/input/code/dkt/train.py", line 36, in main
    trainer.run(args=args, train_data=train_data, valid_data=valid_data, model=model)
  File "/opt/ml/input/code/dkt/dkt/trainer.py", line 43, in run
    train_auc, train_acc, train_loss = train(train_loader=train_loader,
  File "/opt/ml/input/code/dkt/dkt/trainer.py", line 93, in train
    preds = model(batch) # '**'를 사용하기 위해 parameter와 argument의 쌍이 같아햐 하는데 lstm에서 paramete는 data하나기 때문에 '**'안씀
  File "/opt/ml/miniconda3/envs/dkt/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1518, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
  File "/opt/ml/miniconda3/envs/dkt/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1527, in _call_impl
    return forward_call(*args, **kwargs)
  File "/opt/ml/input/code/dkt/dkt/model.py", line 79, in forward
    X, batch_size = super().forward(data) # super().forward은 부모객체의 forward메소드를 말함
  File "/opt/ml/input/code/dkt/dkt/model.py", line 64, in forward
    print(embed.view())
TypeError: view() received an invalid combination of arguments - got (), but expected one of:
 * (torch.dtype dtype)
 * (tuple of ints size)